examples/nodejs/eventbridge/README.md

```diff
@@ -0,0 +1,54 @@
+# Welcome to Momento <-> Eventbridge Integration Example Project
+
+The project demonstrates a write-through cache pattern for DynamoDB using DynamoDB Streams, AWS EventBridge and Momento.
+
+### **Prerequisites:**
+
+- Momento API Key, can be created using [momento console](https://console.gomomento.com/) if you haven’t already created one
+- [Token Vending Machine](https://github.com/momentohq/client-sdk-javascript/tree/main/examples/nodejs/token-vending-machine) deployed to AWS account
```

Comment:
have you looked into how much work it might be to pull the stuff you need for TVM directly into your stack here?

Reply:
It won;t be much work, but for consistency with the vite-chat-app, I kept it this way. The vite-chat app also lies in the nodejs example directory and asks customers to deploy the TVM first. Let me know if it makes sense to keep it this way?

Reply:
actually,

Reply:
if we are going to hard-code AWS creds into the app maybe we should just hard-code the momento key too?

Reply:
that would keep the scope of this demo a little simpler and let people focus on the parts you're trying to highlight.

Reply:
we can chat about this if you like!